templates/document_head.hbs

```diff
@@ -6,5 +6,6 @@
 <meta name="apple-mobile-web-app-title" content="Jobber Help Center">
 <meta name="application-name" content="Jobber Help Center">
 <!-- TODO: FAVICONS -->
-<script async defer src="https://d3ey4dbjkt2f6s.cloudfront.net/source_tracking.js" type="text/javascript"></script>
+<script async defer src="https://cdnjs.cloudflare.com/ajax/libs/magnific-popup.js/1.1.0/jquery.magnific-popup.min.js"></script>
```

Comment:
scritp is still in even though the popup was removed

Reply:
removed!